indexer/editable_map_object.cpp

```diff
@@ -232,12 +232,14 @@ bool EditableMapObject::ValidateHouseNumber(string const & houseNumber)
 // static
 bool EditableMapObject::ValidateFlats(string const & flats)
 {
-  auto it = strings::SimpleTokenizer(flats, ";");
-  for (; it != strings::SimpleTokenizer(); ++it)
+  for (auto it = strings::SimpleTokenizer(flats, ";"); it; ++it)
   {
     auto token = *it;
     strings::Trim(token);
-    vector<string> range(strings::SimpleTokenizer(token, "-"), strings::SimpleTokenizer());
+
+    vector<string> range;
+    for (auto i = strings::SimpleTokenizer(token, "-"); i; ++i)
```

Comment:
Кто решил? Когда решил? Тот Split что в соседнем PR я не уверен что нужен в
base, его бы перенести поближе к месту использования.
On Jun 22, 2016 10:17 AM, "mpimenov" <notifications@github.com> wrote:

> In indexer/editable_map_object.cpp
> <https://github.com/mapsme/omim/pull/3627#discussion_r68004575>:
>
> >    {
> >      auto token = *it;
> >      strings::Trim(token);
> > -    vector<string> range(strings::SimpleTokenizer(token, "-"), strings::SimpleTokenizer());
> > +
> > +    vector<string> range;
> > +    for (auto i = strings::SimpleTokenizer(token, "-"); i; ++i)
>
> Это ведь Split, который всё равно решили добавлять в base?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mapsme/omim/pull/3627/files/3dcca2c16d45bd7e45fc8a8c6d4e519015d5acc7#r68004575>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAbvp8bTtgcrb0qvlwoABSWtbfiICfkcks5qOOGVgaJpZM4I7Lb1>
> .
>